pgmq-extension/test/sql/base.sql

```diff
@@ -29,6 +29,9 @@ SELECT * from pgmq.send('test_default_queue', '{"hello": "world"}');
 \set msg_id 1
 SELECT msg_id = :msg_id FROM pgmq.read('test_default_queue', 2, 1);
 
+-- read message using conditional
+SELECT msg_id = :msg_id FROM pgmq.read('test_default_queue', 2, 1, '{"hello": "world"}');
+
```

Comment:
@Neptune650 , maybe this test fails because the statement above sets VT to 2 seconds, which means the message is probably invisible when we try to read on this line?

Reply:
You were correct about this, thanks.